front/app/containers/InitiativesIndexPage/index.tsx

```diff
@@ -71,6 +79,17 @@ const Padding = styled.div`
 interface Props {}
 
 const InitiativeIndexPage = memo<Props>(() => {
+  const initiativesEnabled = useSettingEnabled('initiatives');
+  const locale = useLocale();
+
+  if (initiativesEnabled === null || isNilOrError(locale)) return null;
+
+  if (!initiativesEnabled) {
+    clHistory.replace(`/${locale}/*`);
```

Comment:
Not sure if I understand completely here (how would this redirect to 'Page not found'?), but I would actually keep this as a separate task outside of this epic. It's a larger problem we need to tackle platform-wide.

So if you keep it, I would add some comments and/or simplify the render conditions, but would definitely log it in the TEC board .

Reply:
Added a comment, will also add it to the technical debt :+1: